selfdrive/car/hyundai/values.py

```diff
@@ -35,6 +35,11 @@ def __init__(self, CP):
       self.STEER_DELTA_UP = 2
```

Comment:
can't make a note any higher up, but note that we found the driver torque signal on the EV6 to be much greater magnitude than our Sonata, so we needed to allow more allowance to compensate. is this true for this? you can check by comparing lateral accel for certain driver torques when disabled